kubectl-fdb/cmd/profile_analyzer.go

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
Please add a test case for this new sub command. We also should add additional documentation on how to use the new subcommand.